README.md

```diff
@@ -19,29 +20,30 @@ Inside of Forza you are required to setup a few different settings to spit out t
 | Forza Horizon 5    | 9925 |
 
 ### Network Isolation ###
+
 Steam/Linux players will not have to do this but for Xbox / Windows store owners, Windows Apps prevent localhost connections by default for "security" This will need to be disabled for this program to work
 
 Thankfully it can be done on a per-app basis by running the corresponding powershell command for your game:
-#### Forza Motorsport 7: ####
 
-```
-CheckNetIsolation.exe LoopbackExempt -a -n="microsoft.apollobasegame_1.174.4791.2_x64__8wekyb3d8bbwe"
+#### Forza Motorsport 7 ####
+
+```cmd
+CheckNetIsolation.exe LoopbackExempt -a -n="microsoft.apollobasegame`_`1.174.4791.2_x64__8wekyb3d8bbwe"

```

Comment:
Why has \` been added around the game name, this should be left untouched.

Reply:
I  don't know, I think linting error

Reply:
fixed